NCRO documentation fixes

rosehartman opened the pull request:
I fixed the NCRO documentation to make sure all variables were included and added a citation for NCRO to the Readme. I also updated the Readme with the new location of the repo, but I'm not 100% sure I did that part right. 

I didn't add the "=" to the "sign" variables that didn't have values for the variables themselves, since I think we're still trying to figure that out.

sbashevkin commented on data-raw/NCRO.R:
```diff
@@ -73,7 +77,7 @@ NCRO <- mutate(NCROX,
                 Datetime = with_tz(mdy_hm(`Collection Date`, tz = "Etc/GMT+8"), tzone = "America/Los_Angeles"),
                 Date = date(Datetime)) %>%
-  filter(`Sample Type` == "Normal Sample") %>%
+  filter(`Sample Type` == "Normal Sample", !is.na(sign)) %>%
```
What is the effect of this? Does it just remove rows with NA values in a variable? And does it only remove values for that variable since it's in long format, or does it impact values in other variables as well?

rosehartman commented:
Ah, That actually does nothing. I had added that earlier while trying to figure out which 'sign' variables had NA's and why. It turns out the NA's are created during the pivot_wider step and weren't in the 'long' format data at all, so nothing got filtered out in that step. I just removed it.
